asset-transfer-sbe/chaincode-java/src/main/java/org/hyperledger/fabric/samples/sbe/AssetContract.java

```diff
@@ -185,12 +206,46 @@ private void setAssetStateBasedEndorsement(final Context ctx, final String asset
     }
 
     /**
-     * Retrieves the client's OrgId (MSPID)
+     * Sets an endorsement policy to the assetId Key.
+     * Enforces that any N out of the Org's Peers specified must endorse future update transactions for the specified assetId Key.
      *
      * @param ctx the transaction context
-     * @return String value of the Org MSPID
+     * @param assetId the id of the asset
+     * @param nOrgs the number of N Orgs to endorse out of the list of Orgs provided
+     * @param ownerOrgs the list of Owner Org MSPID's
      */
-    private String getClientOrgId(final Context ctx) {
-        return ctx.getClientIdentity().getMSPID();
+    private void setAssetStateBasedEndorsementWithNOutOfPolicy(final Context ctx, final String assetId, final int nOrgs, final String[] ownerOrgs) {
+        StateBasedEndorsement stateBasedEndorsement = StateBasedEndorsementFactory.getInstance().newStateBasedEndorsement(null);
+        stateBasedEndorsement.addOrgs(StateBasedEndorsement.RoleType.RoleTypeMember, ownerOrgs);
```

Comment:
LGTM

minor : line 218/219 is not really needed in this func, since you can pass the MspIds directly to this.policy()

Reply:
done, and thanks for this, we donot need it anymore :)